database/guests.ts

```diff
@@ -1,3 +1,5 @@
+import { sql } from './connect';
+
 declare module globalThis {
   let guestsDatabase: Guest[];
 }
```

Comment:
Maybe we don't need this `globalThis` and related code anymore here?

Reply:
Thx for the catch. Forgot to remove it when changing to DB queries! [826cb1b](https://github.com/upleveled/expo-example-spring-2024-atvie/pull/4/commits/826cb1b5fbb5160cc1372fe76fad88cddb9b3daf)